src/components/_buttons/ConnectButton/ConnectWalletPopover.tsx

```
@@ -1,201 +1,58 @@
-              )
-            })}
-          </Stack>
-        </PopoverBody>
-      </PopoverContent>
-    </Popover>
-  )
-}
+      </ConnectButton.Custom>
+    )
+  }
+)
```

ESLint

[error] Component definition is missing display name

(react/display-name)